src/test/resources/pages/instructorFeedbackConstSumOptionQuestionAddSuccess.html

```diff
@@ -203,9 +203,13 @@ <h5 class="col-sm-2 margin-top-0">
                   </label>
                 </h5>
                 <div class="col-sm-10">
-                  <textarea class="form-control" cols="100%" disabled="" id="instructions" name="instructions" placeholder="e.g. Please answer all the given questions." rows="4">
-                    Instructions for first session
-                  </textarea>
+                  <div class="panel panel-default panel-body mce-content-body" id="instructions" spellcheck="false">
+                    <p>
+                      Instructions for first session
+                    </p>
+                  </div>
+                  <input name="instructions" type="hidden">
+                  <input name="instructions" type="hidden">
```

Comment:
When tinyMCE works in an inline mode, it sends value via hidden input. As I remember, duplicate appears after reinitilizing that is done when we choose edit/readonly mode while editing feedback session. I didn't notice any side-effects of duplicate inputs while testing